os/sys/rtimer.h

```diff
@@ -189,14 +194,24 @@ void rtimer_arch_schedule(rtimer_clock_t t);
 #endif /* RTIMER_CONF_GUARD_TIME */
 
 /** \brief Busy-wait until a condition. Start time is t0, max wait time is max_time */
+#if CONTIKI_TARGET_COOJA
```

Comment:
I would suggest `#ifndef RTIMER_BUSYWAIT_UNTIL_ABS` here. And then the cooja-specific stuff can go in a cooja-specific header somewhere.